modules/cassandra/src/main/resources/cassandra-configuration-example/cassandra.yaml

```diff
@@ -0,0 +1,1233 @@
+# Cassandra storage config YAML
```

Comment:
IMO we should remove the commented options to make it easier to understand what config we use to run C* by default

Reply:
By default Cassandra container does nothing with Cassandra config and doesn't have any default config. It just uses config from Docker container (unlike of Maria db testcontainer) as is. This file is only example of configuration. Btw, I added comment line there to clear things out.

Reply:
If it's unused, we definitely should not put it in `src/main/resources`. 
If user decides to override the config he most probably knows its structure and examples

Reply:
As I said - it's only an example. As mentioned in top of the file and also in packet naming. 
>probably

It's only a guess. I believe that more information is better. Ok, if you really want, I removed this packet completely.